tetris.js

```diff
@@ -132,29 +132,49 @@ module.exports = {
 	buildTetramino: function(coordinats){
 		let math = coordinats.split("\n")
 		let arrCoordinats = []
+		let quantityElem = 0;
 		
 		for(let i = 0, elem_Y = 0; i < math.length; i++){
 			if(math[i]!= false){
+				
 				for(let j = 0, elem_X = 0; j < math[i].length; j++){
 					let obj = {}
-					if(math[i][j] == "X"){
-						console.log(math[i][j])
-						obj.x = elem_X
-						obj.y = elem_Y
-						elem_Y++
-						
-						arrCoordinats.push(obj)
-						
-					}
 					if(math[i][j]!= false){
+						if(math[i][j] == "X"){
+							console.log(math[i][j])
+							obj.x = elem_X
+							obj.y = elem_Y
+							quantityElem++;
+							
+							arrCoordinats.push(obj)
+							
+						}
 						elem_X++
 					}
+					
 				}
+				elem_Y++
 			}
 			
 		}
-		
-		console.log(arrCoordinats)
+		let condition = true;
```

Comment:
maybe describe condition in the name of variable instead of saying just "condition"